webapp/app/js/directives/clonejobs.js

```diff
@@ -236,7 +236,10 @@ treeherder.directive('thCloneJobs', [
             jobStatus.key = key;
             if(parseInt(job.failure_classification_id, 10) > 1){
                 jobStatus.value = job.job_type_symbol + '*';
-            }else{
+                if (jobStatus.btnClassClassified) {
+                    jobStatus.btnClass = jobStatus.btnClassClassified;
```

Comment:
I'm presuming we need the conditional because some result states don't have the solid border to start with? Could you add a comment explaining? eg
// For result types that are displayed more prominently when unclassified, switch to the more subtle unclassified style.